src/libs/ajax.js

```diff
@@ -169,8 +169,7 @@ const fetchMartha = withUrlPrefix(`${getConfig().marthaUrlRoot}/`, fetchOk)
 const fetchBard = withUrlPrefix(`${getConfig().bardRoot}/`, fetchOk)
 const fetchEcm = withUrlPrefix(`${getConfig().externalCredsUrlRoot}/`, fetchOk)
 
-const nbName = name => encodeURIComponent(`notebooks/${name}.${tools.Jupyter.ext}`)
-const rName = name => encodeURIComponent(`notebooks/${name}.${tools.RStudio.ext}`)
+const encodeName = name => encodeURIComponent(`notebooks/${name}`)
```

Comment:
Could this maybe be named something like `encodeAnalysisName`? Since this file contains code related to so many different parts of the app, a generic name could be mistaken for something else.